client/src/components/onboarding/ProfileSetting.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect } from "react";
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography'; 
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import DropdownSelect from '../DropdownSelect';
+import moment from 'moment-timezone';
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import { useHistory } from 'react-router-dom';
+
+
+const useStyles = makeStyles({
+
+    body: {
+        padding: "50px 50px 50px 50px",
+        marginBottom:"20px"
+    },
+
+    button: {
+        marginBottom:"70px",
+        color:"#ffffff"
+    },
+});
+
+// Selected timezones only for Canada, otherwise there would be too many
+const timezones = moment.tz.zonesForCountry('CA').map((z) => {
+    return {
+        value:z,
+        label:`${z} (${moment(moment()).tz(z).format("HH:mm")})`
+    }
+});
+
+function ProfileSetting({setters}) {
+
+    const classes = useStyles();
+
+    const history = useHistory();
+
+    const clickHandler = () => {
```

Comment:
Will uncomment in another branch

Reply:
Usually we shouldn't be merging these comments into dev. Might as well remove it here since it's not doing anything